modules/robin-map/1.4.0/overlay/BUILD.bazel

```diff
@@ -0,0 +1,17 @@
+load("@rules_cc//cc:cc_library.bzl", "cc_library")
+
+config_setting(
+    name = "msvc_compiler",
+    flag_values = {"@bazel_tools//tools/cpp:compiler": "msvc-cl"},
+)
+
+cc_library(
+    name = "robin-map",
+    hdrs = glob(["include/tsl/*.h"]),
+    copts = select({
+        ":msvc_compiler": ["/std:c++17"],
```

Comment:
Use https://github.com/bazelbuild/rules_cc/blob/99a85777cfdb897e3ea2de51ecd78774f6cfadab/cc/compiler/BUILD#L73

Reply:
sorry, was set to automerge, i will fix the other pr